src/main.rs

```diff
@@ -51,7 +50,11 @@ async fn scan_trackers(central: &Adapter) -> Vec<Peripheral> {
 }
 
 async fn tracker_worker(tracker: &Peripheral) {
-    tracker.connect().await.unwrap();
+    let mut res = tracker.connect().await;
+    while res.is_err() {
+        println!("Failed to connect to tracker, trying again: {:?}", res);
+        res = tracker.connect().await;
+    }
```

Comment:
こんなところでMutable使わないでください ＞＜

```suggestion
loop {
    match tracker.connect().await {
        Ok(_) => break,
        Err(e) => println!("Failed to connect to tracker, trying again: {e:?}"),
    }
};
```